src/utils/fs.ts

```diff
@@ -120,10 +120,10 @@ export async function readPathsRecursively(dir: string): Promise<string[]> {
  * Copies a file from one location to another, it also creates the destination
  * directory if it doesn't exist
  *
- * @param sourceFile Original file path.
  * @param destFile Destination for the file.
+ * @param sourceFile Original file path.
  */
-export async function copyFileWithDir(sourceFile: string, destFile: string) {
+export async function copyFileWithDir(destFile: string, sourceFile: string) {
```

Comment:
no 😭 , sorry I was asking the opposite, to change the `copyVercelStaticAssets` function oredering, not the order of these 😢 

as you can see now this function uses a different ordering compared to the standard nodejs one (like 128👇)

(yeah sorry for being annoying on this one but having different orders I think can really trip people, uhm... me, up 😅 😓)

Reply:
OHHHHH. I'll take the blame for that one, and I'll hide behind the excuse of it being late at night 🙈🙈

Reply:
ahah no problem 😄